### PR TITLE
Adding week 12 blog post

### PR DESCRIPTION
**Pull request by xtanion**

### CODE REVIEW

File: docs/source/posts/2022/2022-09-08-week-12-shivam.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Week 12 - Adding skeletaton as actors and fix global transformation
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:1]: typo `skeleton`

File: docs/source/posts/2022/2022-09-08-week-12-shivam.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+- I started this week by fixing the segmentation fault for the skinning test. I could not fix this as of now.
+
+- I imported the ``RigggedFigure`` model into the blender and I was quite impressed with how it visualizes each bone. So, I created a function that creates an arrow actor in place of a bone, and by applying the correct transformation, We get the bones of a model in place. Here's a quick preview of bones in Blender vs bones in FURY:
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:14]: typo `RiggedFigure` instead

Comment (ganimtron-10) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:14]: `... transformation, we get ...`

Comment (ganimtron-10) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:14]: to keep things consistent, either use `blender` or `Blender`

File: docs/source/posts/2022/2022-09-08-week-12-shivam.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+.. image:: https://user-images.githubusercontent.com/74976752/189194609-e55f6285-b5ed-4eb3-9e78-5fb462fb2dee.png
+    :width: 500
+    :align: center
+
+.. image:: https://user-images.githubusercontent.com/74976752/189195853-5b1f8945-9822-48f5-8d55-f13e822a43a7.png
+    :width: 500
+    :align: center
+
+- After having the bones actor, I noticed that some bones are not aligned correctly, I was happening to bones due to multiplication of the same matrix twice while calculating.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:24]: `... correctly, it was ...`

Comment (ganimtron-10) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:24]: can you also rephrase the last sentence after the comma `,` ??

File: docs/source/posts/2022/2022-09-08-week-12-shivam.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+- I also created a function that calculates the total transformation of a node, It eliminates the need to use ``timeline.get_value`` in ``get_skin_timeline`` function.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:26]: `it` instead

File: docs/source/posts/2022/2022-09-08-week-12-shivam.rst [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+
+What is coming up next week?
+----------------------------
+
+- Tho, it seems like I fixed everything, but the we are not getting the correct model at timestamp 0. We need to find the cause and fix it!
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:32]: remove `the` after `but`

**Review by ganimtron-10 — COMMENTED** [review]
Hello @xtanion , PTAL at my below comments.
Thanks!

File: docs/source/posts/2022/2022-09-08-week-12-shivam.rst [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+.. image:: https://user-images.githubusercontent.com/74976752/189194609-e55f6285-b5ed-4eb3-9e78-5fb462fb2dee.png
+    :width: 500
+    :align: center
+
+.. image:: https://user-images.githubusercontent.com/74976752/189195853-5b1f8945-9822-48f5-8d55-f13e822a43a7.png
+    :width: 500
+    :align: center
+
+- After having the bones actor, I noticed that some bones are not aligned correctly, it was happening due to multiplication of the same transformation matrix twice.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:24]: I think `..., it was` should be a new sentence `. It was`

**Review by m-agour — COMMENTED** [review]
Hi @xtanion 
It's a bad sign starting your week and your blog post with the error that shall not be named :smile:. But anyway, I only have one comment below, thanks.

File: docs/source/posts/2022/2022-09-08-week-12-shivam.rst [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+- I also created a function that calculates the total transformation of a node, it eliminates the need to use ``timeline.get_value`` in ``get_skin_timeline`` function.
+
+
+What is coming up next week?
+----------------------------
+
+- Tho, it seems like I fixed everything, but we are not getting the correct model at timestamp 0. We need to find the cause and fix it!
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:32]: `Tho` ? avoid slang

File: docs/source/posts/2022/2022-09-08-week-12-shivam.rst [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+- Cleaning the Simple Animation PR `#643`_, and merging it.
+
+
+Did you get stuck anywhere?
+---------------------------
+
+- While applying the timeline, we were getting the identity matrix for timestamp 0.0s, tho it's set to a new value before. We figured this in our meeting that it's happening due to some model's animation not starting from 0.0s.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-09-08-week-12-shivam.rst:40]: tho ?

**Review by skoudoro — COMMENTED** [review]
Hi @xtanion, 

See below my comments

**Comment by skoudoro** [comment]
Thank you for updating, merging

**Pull request merged**